CONTRIBUTING.md

```diff
@@ -94,6 +94,19 @@ Here's what the contribution process looks like:
 
 > ⚛️ For RNW-specific walkthrough on setting up your branch, see the [Setup page](https://github.com/microsoft/react-native-windows/wiki/Setup) in our Wiki. 
 
+### 🔙 **Backporting**
+There are cases when we'll want to port a fix back into one or more of the previous stable releases. If you are working on backporting a fix, your development process will look similar to what's described above, main difference being that your feature branch will need to build off of one of the n-stable branches as the base, instead of main. For example, if you were backporting a change to 0.67-stable you would run the following commands to get your branch set up:
+
+```
+> git fetch upstream
+> git checkout 0.67-stable
+> git pull upstream 0.67-stable
+> git checkout -b backport_branch_name
+```
+
+Then once you've added and tested the desired changes, you'll have to make sure to specify the target stable branch, instead of main, as the base branch when creating your PR.
+
+> Note that in order to preserve stability and compatibility we don't backport all fixes, and all backport changes will go through additional triage to get approved.
```

Comment:
nit: I recently removed the triage requirement for 0.66+ branches, since some structural changes and extra validation removes the need.

Reply:
Good to know! Will this always be true for n-1 stable branches?